### ISSUE TITLE
add active editor state information to editor's text area aria label

### ISSUE THREAD

**Issue by meganrogge** [microsoft/vscode#242617]
cc @rperez030 and @jooyoungseo

Roberto has mentioned that the way he navigates between editors, the window title is not read by the screen reader. 

Given that active editor state is important information, we're thinking it should be appended to the text area's aria label so that it's read any time the editor is focused.

Any thoughts or feedback would be appreciated. We will also get insights and feedback from users with this added to the title by default in screen reader mode this iteration.

**Comment by meganrogge** [comment]
From @jooyoungseo, It doesn't read the window title when you switch with `ctrl+tab` between editors in an editor group. It does with `alt+tab`.

**Comment by meganrogge** [comment]
We're concerned about adding this to the text area because it could be very noisy.

**Comment by meganrogge** [comment]
We think it should be added, but have a setting to disable it.

**Comment by rperez030** [comment]
> We're concerned about adding this to the text area because it could be very noisy.

I agree, and I suggest being cautious about it. in my opinion, the core issue here is the lack of feedback when a significant context change takes place within the editor, for example, when using copilot edits, so maybe another solution is to simply add announcements for signals that already exist such as "Chat Edit Modified" or "Code action applied" which are currently available via sound only. I think that would reduce the need for this.
